index.html

```diff
@@ -236,80 +236,82 @@ <h1 class="m-4 font-medium text-xl text-black">
+                        data-month="1"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-20 right-6 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="2"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-32 right-4 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="3"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-44 right-8 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="4"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-[215px] right-[60px] text-xl w-10 h-10 rounded-full text-center"
+                        data-month="5"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute bottom-2 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="6"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-56 left-16 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="7"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-44 left-6 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="8"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-32 left-2 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="9"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-20 left-4 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="10"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-10 left-12 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="11"
+                      >
+                        .
+                      </span>
+                      <span
+                        class="dot absolute top-4 text-xl w-10 h-10 rounded-full text-center"
+                        data-month="12"
+                      >
+                        .
+                      </span>
+                    </div>
```

Comment:
The formatting seems weird in this file

Reply:
Its prettier extension
I don't like it either.